tests/integration/helpers/loginByGoogle.js

```diff
@@ -0,0 +1,45 @@
+import { ClientFunction, Role } from "testcafe";
+const fetch = require("node-fetch");
+
+const setWindowAccessToken = ClientFunction((data) =>
+  window.__store__.dispatch("auth/setAccessToken", data)
+);
+
+export const googleAuthUser = Role(
+  `${process.env.BROWSERSTACK_BASE_URL}/login`,
+  async (t) => {
```

Comment:
what is `(t)`?

Reply:
(t) was the testcafe instance that we receive. From your other feedback, I've renamed it.